pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -307,37 +307,49 @@ func ResourceConfigFromReply(reply *lifecycle.EventReply, resources *ResourceCon
 
 // Updates the supplied container config in-place based on the isolation
 // controls in the event reply.
-func UpdateContainerConfigWithReply(reply *lifecycle.EventReply, config *runtime.ContainerConfig) {
+func UpdateContainerConfigWithReply(reply *lifecycle.EventReply, config *runtime.ContainerConfig) *runtime.ContainerConfig {
```

Comment:
@ConnorDoyle Why do you think that ? This function is similar to `ResourceConfigFromReply` where we dont return any error.